dns_server.go

```diff
@@ -130,7 +130,7 @@ func getDnsServerAddr(c *Config) string {
 func startDns(l *logrus.Logger, c *Config) {
 	dnsAddr = getDnsServerAddr(c)
 	dnsServer = &dns.Server{Addr: dnsAddr, Net: "udp"}
-	l.Debugf("Starting DNS responder at %s\n", dnsAddr)
+	l.Infof("Starting DNS responder at %s\n", dnsAddr)
```

Comment:
We should use a `WithField` here, either `udpAddr` or `dnsAddr` will work.

Reply:
Updated to dnsListener, as this tracks with the "sshListener" field name logged for the same purpose.